src/components/layouts/pages/inspect/Inspect.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { getPdfFiles } from '../../../../store/pdfFiles/selectors';
+import connect from 'react-redux/es/connect/connect';
```

Comment:
better use import from library root:
```
import { connect } from 'react-redux';
```

Also we try to keep imports in the following order:
 - 3rd-party imports
 - our js module imports
 - resources like images, stylesheets

So please move our `import { getPdfFiles } from '../../../../store/pdfFiles/selectors';` lower